runtime/errors.go

```diff
@@ -386,3 +386,21 @@ func (e *EnumCaseMismatchError) Error() string {
 		e.foundName,
 	)
 }
+
+// MissingEnumCasesError is reported during an enum update, if any enum cases are removed
+// from an existing enum.
+type MissingEnumCasesError struct {
+	declName string
+	expected int
+	found    int
```

Comment:
All other errors export the fields so maybe do the same here

Reply:
Added in 51bb585